src/main/java/edu/harvard/iq/dataverse/search/SearchIncludeFragment.java

```diff
@@ -338,8 +338,8 @@ public void search(boolean onlyDataRelatedToMe) {
                 logger.info(solrQueryResponse.getError());
                 setSolrErrorEncountered(true);
             }
-            // This 2nd search() is for populating the facets: -- L.A. 
-            // TODO: ...
+            // This 2nd search() is for populating the "type" ("dataverse", "dataset", "file") facets: -- L.A. 
+            // (why exactly do we need it, again?)
```

Comment:
I believe it's to get the counts (since it shows those counts even if that facet is not checked, ie those types are not returned by the search).